two_factor/compat.py

```diff
@@ -11,27 +11,43 @@
     from django.contrib.formtools.wizard.views import SessionWizardView
     from django.contrib.formtools.wizard.storage.session import SessionStorage
 
+try:
+    from django.contrib.sites.shortcuts import get_current_site
+except ImportError:
+    from django.contrib.sites.models import get_current_site
 
-if django.VERSION[:2] >= (1, 8):
```

Comment:
Not a strong reason, just a preference/instinct to use feature detection. I'll add a commit to use version sniffing instead.